Consensus-CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Consensus Changelog
+
+The Consensus packages are organized in two bundles: the packages that are
```

Comment:
If we release each package in the bundles separately, will it not be the case that only `ouroboros-consensus` and `ouroboros-consensus-cardano` have a `CHANGELOG.md` included in the distributed files? Should we think about how we could include the `CHANGELOG.md` file in each package automatically?

Reply:
All I can think of is creating a stub `CHANGELOG.md` which says that the changelog is in some other place, something like:

```md
# CHANGELOG

This package is part of a bundle of packages. Please consult `CHANGELOG.md` in the 
package `ouroboros-consensus` at the same version as this package to see the current 
changelog that applies also to this package.
```